content/blog/culture-war-open-source.md

```diff
@@ -0,0 +1,170 @@
++++
+title = "The Ethical Source Working Group Blog"
+heading = "The Culture War in Open Source is On"
++++
+
+The challenges to open source\'s \"noble neutrality.\"
```

Comment:
No need to escape the single and double quotes

Reply:
Oh, sorry—just removed that line, which doesn't fit the format here. Those were the only escape backslashes I could find. Again, sorry—an excess introduced by pandoc in conversion.

Just committed to the fork.